init-test

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash -e
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$DIR"
+source "$DIR/environment/util.sh"
+
+# this should fix --break-system-packages not being available in older distros
+export PIP_BREAK_SYSTEM_PACKAGES=1
+
+pip install uv
+export PATH="$HOME/.local/bin:$PATH"
+
+function setup_virtualenv () {
+    pushd $1 > /dev/null
+    echo "Setting up virtualenv for: $1"
+
+    # remove old virtualenv
+    if [ -d ve3 ]; then
+        rm -rf ve3
+    fi
+
+    # create new virtualenv using uv
+    uv venv ve3 || exit 1
```

Comment:
Is it better to let it fail without `exit 1`, then you get more comprehensive log output, since this fails faily default, the same goes for the ` uv pip install -r requirements.txt || exit 1`. 

If you get the same output, then my comment is pointless and I am hallucinating :D